lizard_viewer.py

```diff
@@ -242,12 +247,35 @@ def show_data(self, public_or_private):
             "Downloading {}...".format(asset_type))
         # Get a list with JSONs containing the data from the Lizard API
         payload = {"page_size": 100}
+        # Get the selected public or private area
+        if public_or_private is "private":
+            area_combobox = self.dockwidget.area_combobox_private
+        else:
+            area_combobox = self.dockwidget.area_combobox_public
+        area_index = area_combobox.currentIndex()
+        area_type = AREA_FILTERS[area_index]
+        # Add the bbox to the payload if 'Current view' is selected
+        if area_type == "Current view":
```

Comment:
`"Current view"` is still kinda a magic string here, I think... 
Is probably okay for now, but it's maybe an idea to put this section into a function.